tox.ini

```diff
@@ -14,6 +14,13 @@ commands =
 deps =
     -r {toxinidir}/requirements.txt
     -r {toxinidir}/dev-requirements.txt
+whitelist_externals =
+     docker-compose
+     sleep
+commands_pre =
+     docker-compose -f {toxinidir}/tests/spark_container/docker-compose.yml up --detach
```

Comment:
We were not able to use `tox-docker` easily as they do not support as they do not support [commands](https://github.com/tox-dev/tox-docker/issues/91).